Automattic/pocket-casts-ios
# Mini player redesign

Fixes #

This PR updates the design of the mini-player with the following
 - Transparent margins around the player
 - Rounded corners 
 - Improved button sizes.
 - Shadows

A small note regarding shadow on dark mode, the shadow is always black independently of the theme so there is no noticeable shadows. This is by design defined by @david-gonzalez-a8c 

| Light | Dark | Rose | Classic | Indigo |
| - | - | - | - | - |
| ![Simulator Screenshot - iPhone SE (3rd generation) - 2024-04-24 at 18 14 12](https://github.com/Automattic/pocket-casts-ios/assets/651601/3738ac81-d487-4c1e-bf19-5a680911fa4d) | ![Simulator Screenshot - iPhone SE (3rd generation) - 2024-04-24 at 18 08 17](https://github.com/Automattic/pocket-casts-ios/assets/651601/06eb2b09-7f9c-4117-bd70-4a10d208999e) | ![Simulator Screenshot - iPhone SE (3rd generation) - 2024-04-24 at 18 08 31](https://github.com/Automattic/pocket-casts-ios/assets/651601/3a56da13-6d03-4db4-855c-803a0d59ac58) | ![Simulator Screenshot - iPhone SE (3rd generation) - 2024-04-24 at 18 08 43](https://github.com/Automattic/pocket-casts-ios/assets/651601/f64f71ae-82e8-409c-96b2-9adaf0133ef5) |  ![Simulator Screenshot - iPhone SE (3rd generation) - 2024-04-24 at 18 09 00](https://github.com/Automattic/pocket-casts-ios/assets/651601/ea5563ea-88a6-4050-9f86-6aa20c444b73) |


https://github.com/Automattic/pocket-casts-ios/assets/651601/84a4f568-56cb-4806-80ce-ab3e75270916

## To test

 - Start the app
 - Start to play an item and display the mini-player
 - Check the looks on different themes
 - Check the transitions from mini-player to full player by pressing on it

## Checklist
